opentasks/src/main/java/org/dmfs/tasks/ViewTaskActivity.java

```diff
@@ -65,7 +67,7 @@ protected void onCreate(Bundle savedInstanceState)
 
         if (savedInstanceState == null)
         {
-            ViewTaskFragment fragment = ViewTaskFragment.newInstance(getIntent().getData());
+            ViewTaskFragment fragment = ViewTaskFragment.newInstance(getIntent().getData(), new PrimaryColor(this));
```

Comment:
How about passing this color to the activity as well? On my device I see the primary color briefly when opening a task, we could avoid that by passing the color to the `ViewTaskActivity`. Just make sure you tolerate the absence of the color in case it's called by an external app.

Reply:
I've added color as an optional extra to `ViewTaskActivity`.